Packs/Jira/Integrations/JiraV2/JiraV2.yml

```
@@ -14,10 +14,7 @@ configuration:
   defaultvalue: ""
   type: 0
   required: true
-- display: |-
-    ______________ Basic Authentication _____________
-
-    Username
+- display: Username
```

```suggestion
- display: Username (Basich Authentication)
```
what do you think? @ChanochShayner @amshamah419 @bakatzir

---

done